lib/puppet/type/firewall.rb

```diff
@@ -236,24 +236,6 @@
     newvalues(%r{^\d+[[:graph:][:space:]]+$})
   end
 
-  newparam(:onduplicaterulebehaviour) do
```

Comment:
is that actually a bugfix? Well it might fix a bug, but it's also a breaking change, isn't it?

Reply:
Probably yeh